indigo/distribution.yaml

```diff
@@ -11191,11 +11191,19 @@ repositories:
       version: master
     status: maintained
   urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.1
     release:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/ros-gbp/urdfdom_py-release.git
-      version: 0.3.0-2
+      version: 0.3.1-1
+    source:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.1
```

Comment:
A tag doesn't make sense for the source entry... It might make sense for the doc.

You can just close this pr and re-run bloom with the same arguments as before but with `--pull-request-only` added to them.